dice/.well-known/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
@RoiEXLab After further research, it appears the _.well-known_ directory is an artifact created by Nginx because we're hosting the _dice_ folder as the root of the subdomain `dice.tripleawarclub.org`.  Can you confirm?  (There's a similar folder in the parent directory, which is the root of the `www.tripleawarclub.org` subdomain.)

If so, then there's no reason to track that in this repo, and I'll remove it.

Reply:
I believe the well-known folder is an artifact of the let's encrypt authentication process, and therefore generated automatically, so indeed not needed.

Reply:
Thanks.  Will remove in a follow-up PR.